src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java

```diff
@@ -0,0 +1,124 @@
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
+
+/* Config:
+ * allowSamelineSingleParameterlessAnnotation = true
+ * allowSamelineParameterizedAnnotation = true
+ * allowSamelineParameterizedAnnotation = true
+ */
+
+@MyAnn_22 @com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation.MyAnn_2 // ok
```

Comment:
try to make it `// OK` and not `// ok` {OK in caps thats how it should be}
and also the issue u have mentioned in PR description is already closed
if a separate issue for AnnotationLocationCheckTest is not there u can always make a new issue and work on it. but 1st check if it really not there

Reply:
> and also the issue u have mentioned in PR description is already closed

Ohh mistakenly I pasted a different PR. Its now updated

Reply:
> try to make it `// OK` and not `// ok` {OK in caps thats how it should be}

Ok i will try it. But when I am running the checks in local it is showing we should add `// ok ` not `// OK`

Reply:
your 1 failing check so to know what's wrong click on details and read it, you will find your problem quickly